From 9e2431361660f238ac8d040e674938f6e69acd7e Mon Sep 17 00:00:00 2001
From: Azimjon Ulmasov <azimjon.ulmasov@chainguard.dev>
Date: Tue, 17 Feb 2026 10:09:43 -0500
Subject: [PATCH] Patch out windows and mac tarball creation

Remove bundled JDK builds for Windows and macOS from archives task.
Only build Linux tar.gz without JDK.
Change license to APACHE-LICENSE-2.0 for OSS builds.
---
 rakelib/artifacts.rake | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/rakelib/artifacts.rake b/rakelib/artifacts.rake
index 0003c4d..4fedba1 100644
--- a/rakelib/artifacts.rake
+++ b/rakelib/artifacts.rake
@@ -179,17 +179,10 @@ namespace "artifact" do
   desc "Build all (jdk bundled and not) tar.gz and zip of default logstash plugins with all dependencies"
   task "archives" => ["prepare", "generate_build_metadata"] do
     #with bundled JDKs
-    license_details = ['ELASTIC-LICENSE']
-    @bundles_jdk = true
-    if ARCH == "x86_64"
-      create_archive_pack(license_details, "x86_64", "linux", "windows", "darwin")
-    else
-      create_archive_pack(license_details, "arm64", "linux", "darwin")
-    end
     #without JDK
+    license_details = ['APACHE-LICENSE-2.0', "-oss", oss_exclude_paths]
     @bundles_jdk = false
     build_tar(*license_details, platform: '-no-jdk')
-    build_zip(*license_details, platform: '-no-jdk')
   end
 
   desc "Build jdk bundled tar.gz of default logstash plugins with all dependencies for docker"
-- 
2.49.0

